app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdUpdate.kt

```diff
@@ -43,6 +50,7 @@ class ScanSimpleIdUpdate : Update<ScanSimpleIdModel, ScanSimpleIdEvent, ScanSimp
       val identifier = Identifier(bpPassportCode.toString(), BpPassport)
       next(model = model.searching(), SearchPatientByIdentifier(identifier))
     } catch (e: IllegalArgumentException) {
+      crashReporter.report(e)
```

Comment:
This should ideally be reported as an effect because it makes the `update()` function impure. I'll approve it for now since it's urgent, but we should fix it in a later PR.

Reply:
Yes, this seemed like the easiest approach to save some time. Will do this correctly after the hotfix.